<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="buttonFolder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>1041</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabInputData">
       <attribute name="title">
        <string>Общие исходные данные</string>
       </attribute>
       <widget class="QLabel" name="labelCode">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Нормы</string>
        </property>
       </widget>
       <widget class="QComboBox" name="boxCode">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>СП 52-101-2003</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>СП 63.13330.2012</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="labelTypeSolve">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Тип расчета</string>
        </property>
       </widget>
       <widget class="QComboBox" name="boxTypeSolve">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Проверка</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="labelCode_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>310</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя элемента</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTypeSection">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Тип сечения</string>
        </property>
       </widget>
       <widget class="QComboBox" name="boxTypeSection">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Стандратное сечение</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="textName">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>310</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxD">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>140</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Учет прогиба</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxPS1">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Прочность</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxPS2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>160</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Трещиностойкость</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabStandartSection">
       <attribute name="title">
        <string>Стандартное сечение</string>
       </attribute>
       <widget class="QComboBox" name="boxCode_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Прямоугольник</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="labelCode_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Тип сечения</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelCode_4">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>20</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Армирование</string>
        </property>
       </widget>
       <widget class="QComboBox" name="boxCode_3">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>30</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Прямоугольник</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Двутавр</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Круг</string>
         </property>
        </item>
       </widget>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>70</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="MatplotlibWidget" name="mplwidget">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>40</y>
          <width>400</width>
          <height>300</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="boxCodeConc">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>330</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelCodeConcrete">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>330</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Бетон</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelCode_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>360</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>ybi</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="textName_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>370</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelCodeReinf">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>330</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Арматура</string>
        </property>
       </widget>
       <widget class="QComboBox" name="boxCodeRein">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>330</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelCode_7">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>360</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>ysi</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="textName_3">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>370</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>370</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Нарисовать</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabSolve">
       <attribute name="title">
        <string>Расчет</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>281</width>
          <height>491</height>
         </rect>
        </property>
        <property name="title">
         <string>Расчет по прочности</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>10</y>
          <width>331</width>
          <height>491</height>
         </rect>
        </property>
        <property name="title">
         <string>Расчет по трещиностойкости</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>331</width>
          <height>491</height>
         </rect>
        </property>
        <property name="title">
         <string>Расчет с учетом погибов</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabLoad">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Расчетные нагрузки</string>
       </attribute>
       <widget class="QTableWidget" name="tableLoad">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>741</width>
          <height>471</height>
         </rect>
        </property>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>4</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>5</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>6</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>N, кг</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Mx, кг*м</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>My, кг*м</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Qx, кг</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Qy, кг</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>P, кг</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>b, см</string>
         </property>
        </column>
       </widget>
       <widget class="QSpinBox" name="boxCountLoad">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabNLoad">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Нормативные нагрузки</string>
       </attribute>
       <widget class="QTableWidget" name="tableLoad_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>741</width>
          <height>471</height>
         </rect>
        </property>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>4</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>5</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>6</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>N, кг</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Mx, кг*м</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>My, кг*м</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Qx, кг</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Qy, кг</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>P, кг</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>b, см</string>
         </property>
        </column>
       </widget>
       <widget class="QSpinBox" name="boxCountLoad_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabOutputData">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Вывод по значениям</string>
       </attribute>
       <widget class="QTableWidget" name="tableOutLoad">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1041</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabGeneralOutputData">
       <attribute name="title">
        <string>Общий вывод</string>
       </attribute>
       <widget class="QTableWidget" name="tableOutK">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>441</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableOutGeneral">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>0</y>
          <width>441</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonSolve">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>640</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonWord">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>640</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>В Word</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>640</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelComment">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>640</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listFiles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textFolder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>660</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>621</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>`</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
